tests/system-tests/internal/supporttools/tcpdump.go

```diff
@@ -0,0 +1,539 @@
+package supporttools
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/openshift-kni/eco-gotests/tests/system-tests/internal/await"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/openshift-kni/eco-gotests/tests/system-tests/internal/apiobjectshelper"
+
+	"github.com/golang/glog"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	"github.com/openshift-kni/eco-goinfra/pkg/deployment"
+	"github.com/openshift-kni/eco-goinfra/pkg/pod"
+	scc "github.com/openshift-kni/eco-gotests/tests/system-tests/internal/scc"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+const (
+	logScanMaxTries = 3
+)
+
+// CreateTCPDumpDeployment creates tcpdump deployment on the nodes specified in scheduleOnNodes.
+func CreateTCPDumpDeployment(
+	apiClient *clients.Settings,
+	tdNamespace,
+	tdDeploymentLabel,
+	tdImage,
+	packetCaptureInterface,
+	captureScript string,
+	scheduleOnNodes []string) (*deployment.Builder, error) {
+	glog.V(100).Infof("Getting the image and tag for support-tools from the release image")
```

Comment:
is this still relevant? seems like and image is passed to the method as _tdImage_ parameter